mindmeld/components/dialogue.py

```diff
@@ -1417,14 +1454,15 @@ async def _process_async(self, text, params=None):
                 which may have been set.
 
         Returns:
-            (dict): The dictionary Response.
+            (DialogueResponder): The DialogueResponder Response.
```

Comment:
Do you mean to return a dict instead of a DialogueResponder? These are private methods, so developers should not being using these. Instead, methods like `process` call this method which return a dict. Do we still want to change the implementation to dict?